recipes/kitty/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "kitty" %}
+{% set version = "0.14.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kovidgoyal/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 9eb32808425867dd63580718bc1b97bbd9046d4024279837c6132eb1ef37f189
+
+build:
+  number: 0
+  script: |
+    export PKG_CONFIG_PATH="${PREFIX}/lib/pkgconfig:/usr/lib64/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:${PKG_CONFIG_PATH}"
+    export CFLAGS="${CFLAGS} -I ${PREFIX}/include/freetype2 -I ${PREFIX}/include/harfbuzz"
+    {{ PYTHON }} setup.py linux-package --prefix=$PREFIX --update-check-interval=0
+  skip: True  # [win or py2k or osx]
+  ignore_run_exports:
+    - imagemagick
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+    - make
+  host:
+    - python
+    - dbus  # [linux]
+    - xorg-libxcursor  # [linux]
+    - xorg-libxrandr  # [linux]
+    - xorg-libxi  # [linux]
+    - xorg-libxinerama  # [linux]
+    - xorg-xineramaproto  # [linux]
+    - libglu  # [linux]
+    - xorg-glproto  # [linux]
+    - mesalib  # [linux]
+    - libxkbcommon  # [linux]
+    # the following are listed as runtime deps, but seem to be needed at build
+    - harfbuzz >=1.5.0
+    - zlib
+    - libpng
+    - freetype  # [linux]
+    - fontconfig  # [linux]
+    - imagemagick
+    - pygments
+  run:
+    - python
+    - harfbuzz >=1.5.0
+    - zlib
+    - libpng
+    - freetype  # [linux]
+    - fontconfig  # [linux]
+    - imagemagick
+    - pygments
+
+test:
+  commands:
+    - kitty --help
+
+about:
+  home: https://sw.kovidgoyal.net/kitty/
+  license: GPLv3
```

Comment:
We are not enforcing nor it is required but it would be nice to use the SPDX strings for consistence. In this case it would be `GPL-3.0`.